Portfolio.html

```diff
@@ -1,284 +1,312 @@
+      <li><a href="#"><img src="images/Linkedin.jpg" alt="linkedIn_icon" /></a></li>
+      <li><a href="#"><img src="images/github.jpg" alt="github_icon" /></a></li>
+      <li><a href="#"><img src="images/hand_icon.jpg" alt="hand" /></a></li>
+    </ul>
+
+    <ul class="mySkills">
+      <li class="skill_section lang_section">
+        <div class="skill_header lan_header">
+          <h1 class="sec_header Pro_lang">Languages</h1>
+          <img src="images/down_arrow.svg" alt="down-arrow" />
         </div>
 
-         <ul class="langs">
+        <ul class="langs">
           <li class="lan">
             <img src="images/JavaS.svg" alt="javascript">
-            JavaScript</li>
+            JavaScript
+          </li>
           <li class="lan">
             <img src="images/html.svg" alt="html">
-             HTML
-         </li>
+            HTML
+          </li>
           <li class="lan"><img src="images/css.svg" alt="css">CSS</li>
         </ul>
-       </li>
-
-        <li class="skill_section frameWork">
-          <div class="skill_header frame_header">
-            <h1 class="sec_header frameWorks">Frameworks</h1>
-            <img src="images/icon_right.svg" alt="right"/>
-         </div>
-        </li>
-         <li class="hori_line"><hr></li>
-          
-        <li class="skill_section my_skills">
-          <div class="skill_header my_skills">
-            <h1 class="sec_header my_skills">Skills</h1>
-            <img src="images/icon_right.svg" alt="right"/>
-         </div>
-        </li>  
-     </ul>
-      
-    </section>
-  </body>   
+      </li>
+
+      <li class="skill_section frameWork">
+        <div class="skill_header frame_header">
+          <h1 class="sec_header frameWorks">Frameworks</h1>
+          <img src="images/icon_right.svg" alt="right" />
+        </div>
+      </li>
+
+      <li class="hori_line">
+        <hr>
+      </li>
+
+      <li class="skill_section my_skills">
+        <div class="skill_header my_skills">
+          <h1 class="sec_header my_skills">Skills</h1>
+          <img src="images/icon_right.svg" alt="right" />
+        </div>
+      </li>
+    </ul>
+  </section>
+
+  <footer class="contact_form">
+    <h1 class="contact_form_heading">
+      Contact Form
+    </h1>
+
+    <p class="contactForm_info">
+      If you have an application you are
+      interested in developing, a feature that you need built or a
+      project that needs coding.
+      I’d love to help with it
+    </p>
+
+    <form action="https://formspree.io/f/mpzkoegn" method="post">
+      <input type="text" name="user_name" id="user_name" placeholder="Bonke" required maxlength="30" />
+      <input type="email" name="mail" id="mail" placeholder="bonkegcobo@gmail.com" required /><br>
+      <textarea name="msg" id="msg">Hi, I'm learning programming at microverse</textarea>
+      <button class="btn_submit" type="submit">
+          Get in touch
+        </button>
```

Comment:
- [ ] According to the Figma design you selected, please the styles for your button when you hover or click it.
- [ ] [OPTIONAL] It would be nice if for all your buttons you include the next CSS attribute to improve the UX of your webpage when you hover them:
```
button:hover {
  cursor: pointer;
}
```

Reply:
Hey Dante can I get clarity as to what you mean by the next attribute?

Reply:
Sure! For all your buttons like this one: 
![image](https://user-images.githubusercontent.com/60164239/130818897-01cd5b54-b36d-4742-b5c0-441402d627e7.png)

You can add the pointer cursor when you hover these buttons in order to improve the UX (User Experience) in your website :)